modules/replicator/src/main/java/org/apache/ignite/internal/replicator/ReplicaManager.java

```diff
@@ -271,21 +264,16 @@ private void sendReplicaUnavailableErrorResponse(
             NetworkAddress senderAddr,
             @Nullable Long correlationId,
             ReplicaRequest request,
-            HybridTimestamp requestTimestamp) {
-        var traceId = UUID.randomUUID();
-
-        String errorMessage = IgniteStringFormatter.format("Replica is not ready "
-                        + "[replicationGroupId={}, node={}]",
-                request.groupId(), clusterNetSvc.topologyService().localMember());
-
+            HybridTimestamp requestTimestamp
+    ) {
         if (requestTimestamp != null) {
             clusterNetSvc.messagingService().respond(
                     senderAddr,
                     REPLICA_MESSAGES_FACTORY
                             .errorTimestampAwareReplicaResponse()
-                            .errorMessage(errorMessage)
-                            .errorCode(Replicator.REPLICA_UNAVAILABLE_ERR)
-                            .errorTraceId(traceId)
+                            .throwable(new ReplicaUnavailableException(
```

Comment:
Here and bellow, formatting?
```
                            .throwable(
                                    new ReplicaUnavailableException(
                                            request.groupId(),
                                            clusterNetSvc.topologyService().localMember()
                                    )
                            )
```
Up to u, though.

Reply:
Ok, will do